draft-ietf-scitt-architecture.md

```diff
@@ -878,7 +884,7 @@ In particular, Issuers must carefully review the inclusion of private, confident
 
 In some deployments a special role such as an Auditor might require and be given access to both the Transparency Service and related Adjacent Services.
 
-Transparency Services' Append-only logs MAY carry only cryptographic metadata (e.g. a hash), rather than the complete Signed Statement, which does not raise immediate privacy concerns.
+Transparency Services' Verifiable Data Structures MAY carry only cryptographic metadata (e.g. a hash), rather than the complete Signed Statement, which does not raise immediate privacy concerns.
```

Comment:
```suggestion
Transparency Services' can leverage Verifiable Data Structures which only retain cryptographic metadata (e.g. a hash), rather than the complete Signed Statement, as part of a defense in depth approach to maintaining confidentiality.
```

Reply:
No reason for MAY, but I'm not attached to this change

Reply:
@OR13 "Transparency Services" then?

Reply:
happy for that to be committed